client/src/pages/UserPage.js

```diff
@@ -0,0 +1,101 @@
+import { useState } from 'react';
+
+export default function NewList() {
+  const [isLoading, setIsLoading] = useState();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const form = event.target;
+    const data = {
+      category: form.category.value,
+      brand: form.brand.value,
+      name: form.name.value,
+      description: form.description.value,
+      price: form.price.value,
+      size: form.size.value,
+      condition: form.condition.value,
+      fileName: form.fileInput.files[0].name,
+    };
+
+    uploadFile(data);
+  }
+
+  async function uploadFile(newForm) {
+    const req = {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${localStorage.getItem('token')}`,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(newForm),
+    };
```

Comment:
This isn't the proper way to prepare a request when you are doing a file upload. Isn't your goal to let the user upload their own photo? See the react-file-uploads exercise for the proper way to handle a file upload.